pkg/sdkv2/resources/service_token_acceptance_test.go

```diff
@@ -13,6 +13,7 @@ import (
 )
 
 func TestAccDbtCloudServiceTokenResource(t *testing.T) {
+	t.Skip("Test deprecated")
```

Comment:
Yes, you are right. I skipped this so that I could keep the file around while I ported the tests. This can be removed as it has been moved wholesale into the framework.